the-news/src/App.js

```diff
@@ -1,13 +1,29 @@
 import './App.css';
-import ArticleList from './components/ArticlesList';
-import Header from "./components/Header"
+import Home from './components/Routes/Home'
+import Header from './components/Header'
+import TopicCooking from './components/Routes/TopicCooking';
+import TopicCoding from './components/Routes/TopicCoding';
+import TopicFootball from './components/Routes/TopicFootball';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import TopicsList from './components/TopicsList'
+import NavBar from './components/NavBar';
+
 
 function App() {
   return (
-    <div className="parent">
-      <Header />
-      <ArticleList />
-    </div>
+    <BrowserRouter>
+      <div className="parent">
+        <Header />
+        <NavBar />
+        <Routes>
+          <Route path='/' element={<Home />}/>
+          <Route path='/topics' element={<TopicsList />}/>
+          <Route path='/topics/cooking' element={<TopicCooking />}/>
```

Comment:
There is no need to hard code here, what would happen if the BE had more topics added?